templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -7,6 +7,7 @@ contents: |
   ConditionPathExists=!/etc/ignition-machine-config-encapsulated.json
   # This service is used to move a physical NIC into OVS and reconfigure OVS to use the host IP
   Requires=openvswitch.service
+  Requires=node-valid-hostname.service
   Wants=NetworkManager-wait-online.service
```

Comment:
should we also change `NetworkManager-wait-online.service` to `Requires` instead of `Wants`

Reply:
No, if networkamnager-wait-online fails it can be because other NICs failed to DHCP. That does not mean we do not have a default gateway out of the host, and thus we should still run.